scripts/document.sh

```diff
@@ -51,8 +43,19 @@ jazzy \
     --theme ${THEME} \
     --output ${OUTPUT} \
     --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO
+    
+# Link to turf documentation
+TURF_VERSION=$(grep -A 5 Turf Package.resolved | grep version | cut -d '"' -f 4)
+TURF_BASE_URL="https://mapbox.github.io/turf-swift/${TURF_VERSION}"
+TURF_INDEX=$(curl -s "${TURF_BASE_URL}/search.json")
+TURF_STRUCTS=$(echo ${TURF_INDEX} | python -m json.tool | grep -A 2 \"Structs/ | grep parent_name | uniq | cut -d '"' -f 4 | paste -sd "|" -)
```

Comment:
It looks a bit "too magic" and fragile... What do you think, could it be written in a Python one-liner?

Reply:
Definitely not a one-liner, but I extracted post-processing logic into Python script. PTAL